examples/gizmos/3d_gizmos.rs

```diff
@@ -137,6 +137,16 @@ fn draw_example_collection(
         .circle_segments(64);
 
     gizmos.arrow(Vec3::ZERO, Vec3::ONE * 1.5, YELLOW);
+
+    // You can create more complex arrows using the arrow builder.
+    gizmos
+        .arrow(
+            Vec3::from_array([2., 0., 2.]),
+            Vec3::from_array([2., 2., 2.]),
```

Comment:
Just a small nit: Wouldn't `Vec3::new(...)` be more appropriate here?

Reply:
Yep, when I first wrote this, I used from_array just to keep the old code. I'll change this soon.